src/ray/core_worker/lib/golang/go_worker.cpp

```diff
@@ -0,0 +1,328 @@
+#include "go_worker.h"
```

Comment:
could you name this file ".cc" to respect the convention in the rest of the codebase?